### PR TITLE
fix: Add `serviceId` and `serviceType` sections for DID Document creation for swagger [DEV-3125]

### PR DESCRIPTION
**Pull request by abdulla-ashurov**
- [x] Make default DID Document null for simplifying DID creation.
- [x] Add `serviceId` and `serviceType` fields for `application/x-www-form-urlencoded`

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-3125 Add serviceId and serviceType sections for DID Document creation for swagger](https://app.clickup.com/t/6600954/DEV-3125)

File: src/controllers/issuer.ts [hunk 1]
```diff
@@ -255,8 +257,8 @@ export class IssuerController {
 				if (serviceEndpoint) {
 					didDocument.service = [
 						{
-							id: `${didDocument.id}#service-1`,
-							type: 'service-1',
+							id: didDocument.id+"#"+serviceId,
```
Comment (anikitinDSR) [inline src/controllers/issuer.ts:260]: Let's do it in formatting style, like:
`${didDocuemnt.id}#${serviceId}`

File: src/controllers/issuer.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { generateDidDoc, validateSpecCompliantPayload } from '../helpers/helpers
 export class IssuerController {
 	public static createValidator = [
 		check('didDocument')
-			.optional()
+			.optional({nullable: true})
```
Comment (anikitinDSR) [inline src/controllers/issuer.ts:16]: Why do we need it here? As I know we already allowed to pass empty DIDDocument, did we?

**Review by anikitinDSR — CHANGES_REQUESTED** [review]

File: src/controllers/issuer.ts (hunk 1 above)
Comment (abdulla-ashurov) [reply src/controllers/issuer.ts]: Done! ✅

File: src/controllers/issuer.ts (hunk 2 above)
Comment (abdulla-ashurov) [reply src/controllers/issuer.ts]: Done! ✅

**Pull request closed without merging**